src/callsite.jl

```diff
@@ -238,7 +238,7 @@ end
 
 function show_callinfo(limiter, ci::Union{MultiCallInfo, FailedCallInfo, GeneratedCallInfo})
     types = (ci.sig::DataType).parameters
-    ft, tt... = types
+    ft, tt = types[1], types[2:end]
```

Comment:
I'll make a PR to base so that `tt` stays a `SimpleVector` here, but since that probably won't be backported to 1.7, we'll have to use this workaround instead.